hudi-cli/src/main/java/org/apache/hudi/cli/commands/CompactionCommand.java

```diff
@@ -370,7 +370,7 @@ private String printCompaction(HoodieCompactionPlan compactionPlan,
     List<Comparable[]> rows = new ArrayList<>();
     if ((null != compactionPlan) && (null != compactionPlan.getOperations())) {
       for (HoodieCompactionOperation op : compactionPlan.getOperations()) {
-        rows.add(new Comparable[]{op.getPartitionPath(), op.getFileId(), op.getBaseInstantTime(), op.getDataFilePath(),
+        rows.add(new Comparable[]{op.getPartitionPath(), op.getFileId(), op.getBaseInstantTime(), op.getBaseFilePath(),
```

Comment:
cc @bvaradar is this field name serialized with the plan? if so, we can't change it easily

Reply:
Yes, the name is serialized. @pratyakshsharma : Instead of renaming , can you add alias to the field and see if that works ?